tests.py

```diff
@@ -655,8 +656,8 @@ def test_delete_branch(self, testbot, stash_plugin, monkeypatch):
 
 @pytest.fixture
 def github_api(mocker):
+    mocker.patch.object(github, "Github", autospec=True)
```

Comment:
I think it's better mock `GitHub` before initialize `GitHubAPI`, what do you guys think? @gcardozo123